src/internal.c

```diff
@@ -8140,6 +8140,21 @@ static void FreeProcPeerCertArgs(WOLFSSL* ssl, void* pArgs)
     }
 }
 
+#ifdef HAVE_PK_CALLBACKS
+static void InitSigPkCb(WOLFSSL* ssl, struct SignatureCtx* ctx) {
```

Comment:
You should be able to use `SignatureCtx* ctx` here. Also it might be less confusing to call the variable `sigCtx`, which is also consistent with other places.

Reply:
Check